docs/aws.md

```diff
@@ -147,7 +147,7 @@ aws route53 list-hosted-zones | jq '.HostedZones[] | select(.Name=="kubernetes.c
  ```
  aws route53 change-resource-record-sets \
   --hosted-zone-id <parent-zone-id> \
-  --change-batch subdomain.json
+  --change-batch file://subdomain.json
```

Comment:
I'm testing on MacOS, but the behavior doesn't differ by platform according to the docs.